jxbrowser/src/org/opensim/javabrowser/jxBrowserTopComponent.java

```diff
@@ -55,11 +55,23 @@
 
     public jxBrowserTopComponent() {
         initComponents();
-        
+        String useGPU = "On";        
+        String savedGPU = Preferences.userNodeForPackage(TheApp.class).get("GPU Acceleration", useGPU);
+        Preferences.userNodeForPackage(TheApp.class).put("GPU Acceleration", savedGPU);
+        String useLightWeight = "Off";        
+        String savedLightWeight = Preferences.userNodeForPackage(TheApp.class).get("LightWeight Browser", useLightWeight);
+        Preferences.userNodeForPackage(TheApp.class).put("LightWeight Browser", savedLightWeight);
```

Comment:
How will these appear in preferences on the different OSes? Right now, on macOS, the preferences are under the "path" `org.opensim.utils`. Will these preferences be under a different path?